pkgs/by-name/pi/piscope/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  # nativeBuildInputs
+  pkg-config,
+  wrapGAppsHook3,
+  installShellFiles,
+
+  # buildInputs
+  gtk3,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "piscope";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner = "joan2937";
+    repo = "piscope";
+    tag = "V${finalAttrs.version}";
+    hash = "sha256-VDrx/RLSpMhyD64PmdeWVacb9LleHakcy7D6zFxeyhw=";
+  };
+  # Fix FHS paths
+  postConfigure = ''
+    substituteInPlace piscope.c \
+      --replace /usr/share/piscope $out/share/piscope
+  '';
+
+  nativeBuildInputs = [
+    pkg-config
+    wrapGAppsHook3
+    installShellFiles
+  ];
+  buildInputs = [
+    gtk3
+  ];
+  # Upstream's Makefile assumes FHS
+  installPhase = ''
+    runHook preInstall
+
+    installBin piscope
+    install -D -m 0644 piscope.glade $out/share/piscope/piscope.glade
+
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "http://abyz.me.uk/rpi/pigpio/piscope.html";
+    description = "A logic analyser (digital waveform viewer) for the Raspberry";
```

```suggestion
    description = "Logic analyser (digital waveform viewer) for the Raspberry";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

---

Was fixed in:

https://github.com/NixOS/nixpkgs/commit/22b88ba79d8feb71704fa1baccae978cf9b8d1ab

But thanks for noticing!